NUAttribute.js

```diff
@@ -31,7 +31,7 @@ export default class NUAttribute extends NUObject {
             description: obj.description,
             displayName: (!obj.displayName) ? obj.localName : obj.displayName,
             hasChoices: !(obj.choices === null),
-            isEditable: (!obj.isEditable) ? true : obj.isEditable,
+            isEditable: !obj.isReadOnly,
```

Comment:
In the spec APIs there are more options which makes an attribute non-editable: createOnly, transient and readOnly: how are those used?